docs/controllers.md

```diff
@@ -2,23 +2,26 @@
 
 ## Schemas
 
-Support for any model of a MIDI controller can be added by creating its schema, which is represented as a simple `YAML` file.
-
-### All available fields
-
-- `name`: The name of the controller. Cannot be empty. Must be unique among all schemas.
-- `button_value_off`: The number sent by the controller when a button is in `off` state. Should be in the range `[0, 127]`.
-- `button_value_on`: The number sent by the controller when a button is in `on` state. Should be in the range `[0, 127]`.
-- `knob_value_min`: The minimum value sent by the controller when a knob is rotated. Should be in the range `[0, 127]`.
-- `knob_value_max`: The maximum value sent by the controller when a knob is rotated. Should be in the range `[0, 127]`.
-- `buttons`: List of available buttons on the controller. Each of them consists of:
-  - `id`: The ID of the button that the controller sends with every event. Should be in the range `[0, 127]`.
-  - `name`: A user-defined name for the button that helps to differentiate elements. Cannot be empty. No two elements can have the same name.
-- `knobs`: List of available knobs on the controller.  Each of them consists of:
-  - `id`: The ID of the knob that the controller sends with every event. Should be in the range `[0, 127]`.
-  - `name`: A user-defined name for the knob that helps to differentiate elements. Cannot be empty. No two elements can have the same name.
-
-### Example of a valid schema
+Support for any model of a MIDI controller can be added by creating its schema, which is represented as a simple [YAML](https://yaml.org) file.
+
+The YAML file should contain all fields from the Controller class. See [example](#example-of-a-valid-schema).
+
+Built-in schemas are available in [config_files/controllers](https://github.com/midi-app-controller/midi-app-controller/tree/main/config_files/controllers) at GitHub.
```

Comment:
```suggestion
Built-in schemas are available in [config_files/controllers](https://github.com/midi-app-controller/midi-app-controller/tree/main/config_files/controllers) on GitHub.
```

Reply:
It looks like [`on` is more commonly used](https://ell.stackexchange.com/questions/89101/in-on-or-at-github).